packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -356,7 +356,7 @@ export class NamespacedAttribute extends ElementPatchOperation<string> {
   private namespace: InternedString;
 
   static install(dom: DOMHelper, element: Element, namespace: InternedString, name: InternedString, value: string) {
-    if (value !== null) {
+    if (value !== null && element.tagName) {
```

Comment:
check for element should be `element.nodeType === Node.ELEMENT_NODE` and the typing should be `element: Element | DocumentFragment`